tests/lib/managers/collaboration-whitelists-test.js

```diff
@@ -0,0 +1,427 @@
+/**
+ * @fileoverview Collaboration Whitelists Manager Tests
+ */
+'use strict';
+
+// ------------------------------------------------------------------------------
+// Requirements
+// ------------------------------------------------------------------------------
+var sinon = require('sinon'),
+	mockery = require('mockery'),
+	assert = require('chai').assert,
+	leche = require('leche');
+
+var BoxClient = require('../../../lib/box-client');
+
+// ------------------------------------------------------------------------------
+// Helpers
+// ------------------------------------------------------------------------------
+var sandbox = sinon.sandbox.create(),
+	boxClientFake = leche.fake(BoxClient.prototype),
+	CollaborationWhitelists,
+	collaborationWhitelists,
+	testQS = { testQSKey: 'testQSValue' },
+	testParamsWithQs,
+	COLLABORATION_WHITELIST_ID = '1234',
+	USER_COLLABORATION_WHITELIST_ID = '5678',
+	USER_ID = '12345678',
+	MODULE_FILE_PATH = '../../../lib/managers/collaboration-whitelists';
+
+// ------------------------------------------------------------------------------
+// Tests
+// ------------------------------------------------------------------------------
+
+describe('CollaborationWhitelists', function() {
+
+	before(function() {
+		// Enable Mockery
+		mockery.enable({ useCleanCache: true });
+		// Register Mocks
+		mockery.registerAllowable('../util/url-path');
+		mockery.registerAllowable(MODULE_FILE_PATH);
+	});
+
+	beforeEach(function() {
+		testParamsWithQs = {qs: testQS};
+		// Setup File Under Test
+		CollaborationWhitelists = require(MODULE_FILE_PATH);
+		collaborationWhitelists = new CollaborationWhitelists(boxClientFake);
+	});
+
+	afterEach(function() {
+		sandbox.verifyAndRestore();
+		mockery.resetCache();
+	});
+
+	after(function() {
+		mockery.deregisterAll();
+		mockery.disable();
+	});
+
+	describe('create()', function() {
```

Comment:
These will need to be updated when the method names change in the actual implementation